clang/test/SemaSYCL/msvc-fixes.cpp

```diff
@@ -0,0 +1,7 @@
+// RUN: %clang_cc1 -triple x86_64-windows-msvc -fsycl-is-device -fsyntax-only -verify %s
```

Comment:
```suggestion
// RUN: %clang_cc1 -triple spir-unknown-windows-sycldevice -fsycl-is-device -fsyntax-only -verify %s
```
`x86_64-windows-msvc` - triple is not supposed to work and driver do use it.
Please, make this test work for `spir-unknown-windows-sycldevice` triple.